core/EE_Addon.core.php

```diff
@@ -884,4 +891,19 @@ public function after_registration()
     {
         // cricket chirp... cricket chirp...
     }
+
+    /**
+     * @return mixed
```

Comment:
i didn't add a type to the `$pue_slug` when I whipped up this code example, so my IDE used `mixed` for the type here since it didn't know this should be a string

```suggestion
     * @return string
```